tutorial/markdown/aspnet/semantic-kernel/semantic-kernel-tutorial.md

```diff
@@ -0,0 +1,387 @@
+---
+# frontmatter
+path: "/tutorial-csharp-semantic-kernel-vector-search"
+# title and description do not need to be added to markdown, start with H2 (##)
+title: Build Vector Search with Couchbase .NET Semantic Kernel Connector and OpenAI
+short_title: Vector Search with Semantic Kernel
+description:
+  - Build a semantic search application using Couchbase BHIVE vector index with Semantic Kernel.
+  - Learn to use the Couchbase .NET Vector Store Connector for Microsoft Semantic Kernel.
+  - Discover how to generate embeddings with OpenAI and store them in Couchbase.
+  - Perform vector similarity searches with filtering using SQL++ and ANN_DISTANCE.
+content_type: tutorial
+filter: sdk
+technology:
+  - fts
+  - kv
+tags:
+  - Semantic Kernel
+  - OpenAI
+  - Artificial Intelligence
+sdk_language:
+  - csharp
+length: 30 Mins
+---
+
+## Repository Links
+
+- **Connector Repository**: [couchbase-semantic-kernel](https://github.com/Couchbase-Ecosystem/couchbase-semantic-kernel) - The official Couchbase .NET Vector Store Connector for Microsoft Semantic Kernel
+- **This Example**: [CouchbaseVectorSearchDemo](https://github.com/Couchbase-Ecosystem/couchbase-semantic-kernel/tree/Support-Bhive-and-Composite-Index/CouchbaseVectorSearchDemo) - Complete working example demonstrating vector search with Couchbase
+
+## Introduction
+
+This demo showcases the **Semantic Kernel Couchbase connector** - a .NET library that bridges Microsoft's Semantic Kernel framework with Couchbase's vector search capabilities. The connector provides a seamless integration that allows developers to build AI-powered applications using familiar Semantic Kernel abstractions while leveraging Couchbase's vector indexing for high-performance semantic search.
+
+The connector supports three index types:
+- **BHIVE** (Hyperscale Vector Index) - for pure vector search at scale ← *Used in this demo*
+- **Composite Vector Index** - for vector search with heavy scalar filtering
+- **FTS** (Full-Text Search) - for hybrid text + semantic search
+
+This makes the connector ideal for RAG (Retrieval-Augmented Generation) applications, semantic search engines, hybrid search, and recommendation systems.
+
+## Prerequisites
+
+### 1. Couchbase Server Setup
+- **Couchbase Server 8.0+**
+- Local installation or Couchbase Cloud/Capella
+- Bucket with proper read/write permissions
+- Query service enabled for SQL++ operations
+
+### 2. OpenAI API Access
+- **OpenAI API Key** - Get one from: https://platform.openai.com/api-keys
+- Used for generating text embeddings with `text-embedding-ada-002` model
```

Comment:
Any particular reason to use the old embedding model? text-embedding-3-small should be better both from cost & performance perspective